src/xdocs/config_imports.xml

```diff
@@ -989,6 +1003,52 @@ public class InputIllegalImport { }
           which is part of the Checkstyle distribution.
         </p>
 
+        <h4 id="blacklist-example">Example of blacklist mode</h4>
+        <p>To have a <b>blacklist mode</b>, it is required to have disallows inside subpackage and to have allow rule inside parent of the current subpackage to catch classes and packages those are not in the blacklist.</p>
+        <p>In the example below any import from <code>java.util</code> (<code>java.util.List</code>, <code>java.util.Date</code>) package is allowed except <code>java.util.Map</code> inside subpackage <code>com.puppycrawl.tools.checkstyle.filters</code>.</p>
+        <source>
+&lt;import-control pkg=&quot;com.puppycrawl.tools.checkstyle&quot;&gt;
+    &lt;allow pkg=&quot;java.util&quot;/&gt;
+    &lt;subpackage name=&quot;filters&quot; &gt;
+        &lt;disallow class=&quot;java.util.Map&quot;/&gt;
+    &lt;/subpackage&gt;
+&lt;/import-control&gt;
+        </source>
+        <p>
+          In the next example imports <code>java.util.stream.Stream</code> and <code>java.util.stream.Collectors</code> are disallowed inside <code>com.puppycrawl.tools.checkstyle.checks.imports</code> package,
+          but because of <code>&lt;allow pkg=&quot;java.util.stream&quot;/&gt;</code> every import from <code>java.util.stream</code> is allowed except described ones.
+        </p>
+        <source>
+&lt;import-control pkg=&quot;com.puppycrawl.tools.checkstyle.checks&quot;&gt;
+    &lt;allow pkg=&quot;java.util.stream&quot;/&gt;
+    &lt;subpackage name=&quot;imports&quot;&gt;
+        &lt;disallow class=&quot;java.util.stream.Stream&quot;/&gt;
+        &lt;disallow class=&quot;java.util.stream.Collectors&quot;/&gt;
+    &lt;/subpackage&gt;
+&lt;/import-control&gt;
+        </source>
+
+        <source>
+package com.puppycrawl.tools.checkstyle.checks.imports;
+
+import java.util.stream.Stream;     // violation here
+import java.util.stream.Collectors; // violation here
+import java.util.stream.IntStream;
+        </source>
+        <p>
+          In the following example every imports is allowed except classes <code>java.util.Date</code>, <code>java.util.List</code> and package <code>sun</code>.
```

Comment:
`example every imports is allowed except classes` => `example, all imports are allowed except the classes: `